docs/gallery/general/advanced_hdf5_io.py

```diff
@@ -216,6 +216,56 @@
 #   will be ignored as the h5py.Dataset will either be linked to or copied as on write.
 #
 
+####################
+# Parallel IO using MPI
+# -----------
+#
+# The HDF5 storage backend supports parallel I/O using the Message Passing Interface (MPI).
+# Using this feature requires that you install ``hdf5`` and ``h5py`` against an MPI driver, and you
+# install ``mpi4py``. The basic installation of pynwb will not work. Setup can be tricky, and
+# is outside the scope of this tutorial (for now), and the following assumes that you have
+# HDF5 installed in a MPI configuration. Here we:
+#
+# 1. Instantiate a dataset for parallel write
+#
+# 2. Write to that file in parallel using MPI
+#
+# 3. Read from the file in parallel using MPI.
+#
+# .. code-block:: python
+#
+#   from mpi4py import MPI
+#   import numpy as np
+#   from dateutil import tz
+#   from pynwb import NWBHDF5IO, NWBFile, TimeSeries
+#   from datetime import datetime
+#   from hdmf.data_utils import DataChunkIterator
+#
+#   start_time = datetime(2018, 4, 25, 2, 30, 3, tzinfo=tz.gettz('US/Pacific'))
+#   fname = 'test_parallel_pynwb.nwb'
+#   rank = MPI.COMM_WORLD.rank  # The process ID (integer 0-3 for 4-process run)
+#
+#   # instantiate dataset, but do not write any data
```

Comment:
Instead of ``instantiate dataset, but do not write any data`` maybe say something like.
``Create file on one rank. Here we only instantiate the dataset we want to write in parallel but we do not write any data``

Reply:
```suggestion
#   # Create file on one rank. Here we only instantiate the dataset we want to write in parallel but we do not write any data
```